canonical-web-and-design/maas-ui
# refactor(legacy): remove legacy commission

## Done

- Remove node commission from legacy details.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Visit controller details (you'll probably need a sampledata maas for this QA)
- Open your devtools network tab and click on the legacy websocket.
- Use the take action menu to open the test form and submit.
- Check that the websocket message contains an `enable_ssh` boolean and a `testing_scripts` array.

## Fixes

Fixes: #2528.
